solr/core/src/java/org/apache/solr/handler/admin/IndexSizeEstimator.java

```diff
@@ -388,7 +388,7 @@ private void estimateTermVectors(Map<String, Object> result) throws IOException
         if (liveDocs != null && !liveDocs.get(docId)) {
           continue;
         }
-        Fields termVectors = leafReader.getTermVectors(docId);
+        Fields termVectors = leafReader.termVectors().get(docId);
```

Comment:
`leafReader.termVectors()` should be outside for for loop?

Reply:
yep, missed that
will improve it